server/lib/routes/get-fxa-client-configuration.js

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function normalizeUrl(url) {
```

Comment:
> I can deal with this approach. I'd prefer to use "the one true way" universally, but not at the risk of introducing too many changes at once. Iterate and learn.

FWIW, the sticking-point for me there was that it seemed difficult to work out how to sanely allow updates to an existing config, leaving you with a one-shot config that risks leaving people broken in the worst case. I'm sure a sane story exists there, although I doubt we will ever care until it offers operational flexibility for production that we start caring about.